### PR TITLE
create HE-Chain Proposal

### PR DESCRIPTION
**Pull request by mutobui**
### Project Abstract

> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.

### For which grant [level](https://github.com/w3f/Grants-Program#level_slider-levels) are you applying? 
- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $50,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for > $100k Web3 Foundation Council approval)

### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied, renamed ( `project_name.md`) and updated.
- [x] A BTC or Ethereum (DAI/USDT) address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledged the [terms and conditions](https://github.com/w3f/Grants-Program/blob/master/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process).

### CODE REVIEW

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application. Looks really interesting. Something to be aware of, we only support the development of pallets and not the deployment/integration. So, for example unless you are planning to change/improve the evm module, could you remove it from the deliverables? Also I’m not fully sure what pallet_auction means? Furthermore could you explain the functionality of the dao a little bit more?

**Comment by mutobui** [comment]
Hi, thank you for advising. We focus on the build pallets or on-chain runtime logic that we removed EVM to deliverables.

About DAO:
Currently, by design, there are three objects that need to determine through DAO:

1. Autoban mechanism
There are several rules of Empires that players must follow when they join the pool. These are ideas to keep the pool 'as clean as possible' such as ban/kick AFK(away from keyboard) players, ban/kick players on the Spamming Blacklist.

2. The Empire Pool fee
Pool fee is another base idea to keep the pool clean, with a reasonable fee determined by DAO, the Empires Pool can prevent the network from malicious accounts. When many players join the pool, the total network fee charged by the pool can be significant to grow the ecosystem by granting the projects.

3. The number transaction limit of TX-Handler
The idea of TX-Handler is to protect the network from the DDOS, also determining the right number is important because if the number is not appropriate, it's can drive away many goof projects for example. Currently, our Heroes & Empires only need a maximum of 10 TXs/minute but another good PvP game can require 30 TXs/minutes, at this point the limit number is 30 is reasonable for the HE-Chain network.

**Comment by mutobui** [comment]
pallet_auction helps us run parachain on the Kusama network so pallet_auction also removes it from deliverables like EVM.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the update and sorry for the late reply here. I would recommend removing milestone 3 for now. We usually have very high requirements when it comes to bridges and probably would require you to specify this much more detailed (e.g. what is the advantage compared to https://github.com/Snowfork/snowbridge). Also this way the grant would be below 50k and therefore not fall in the level 3 category, see https://github.com/w3f/Grants-Program#level_slider-levels

**Comment by mutobui** [comment]
Thanks for pointing that out and sorry for the lack of research, we would remove milestone 3 to save a lot of effort to create the same thing as Snowfork. We also add end-user test and end-user documents to milestone 2.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks again for the quick reply. I just have one more request, could you also integrate Weights/Benchmarking into your deliveries or could you reduce the price a little bit. Independent of it, I will share the application already with the rest of the team.

**Review by Noc2 — APPROVED** [review]
Thanks for the update. I'm happy to support the project.

File: applications/HE-Chain.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
+# W3F Grant Proposal
+
+> This document will be part of the terms and conditions of your agreement and therefore needs to contain all the required information about the project. Don't remove any of the mandatory parts presented in bold letters or as headlines! Lines starting with a `>` (such as this one) can be removed.
+>
+> See the [Grants Program Process](https://github.com/w3f/Grants-Program/#pencil-process) on how to submit a proposal.
```
Comment (takahser) [inline applications/HE-Chain.md:5]:
```suggestion
```

**Review by takahser — APPROVED** [review]
I'm happy to support the project as well. 👍

**Review by gautamdhameja — APPROVED** [review]

**Comment by mutobui** [comment]
We want to update some info:
- Rename the project from HE-Chain to Aurora Network with the landing page https://aur.network/ 
- Move the main repository to https://github.com/cryptoviet/aurora-node

**Comment by Noc2** [comment]
@mutobui Sounds good. Feel free to create a new PR/amendment.

**Comment by mutobui** [comment]
Please check the new pull request, we update He-Chain as follows:
- Rename He-Chain to Gafi
- Add pallet_proof_mapping to milestone 1
- Add pallet_staking_pool to milestone 1

**Pull request merged**